### PR TITLE
Fix AlexNet and VGG Implementation.

### PR DESCRIPTION
**Pull request by aksrajvanshi**
Addition: Cleaned AlexNet and VGG implementation (Section 7.1-7.2) after the new nightly build which fixed the Dropout problem and the Conv2d changes.

### CODE REVIEW

File: jupyter/d2l-java/chapter_convolutional-modern/vgg.ipynb [hunk 1]
```diff
@@ -363,7 +347,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 13,
```
Comment (stu1130) [inline jupyter/d2l-java/chapter_convolutional-modern/vgg.ipynb:350]: should it be null?

File: jupyter/d2l-java/chapter_convolutional-modern/vgg.ipynb [hunk 2]
```diff
@@ -396,7 +380,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 14,
```
Comment (stu1130) [inline jupyter/d2l-java/chapter_convolutional-modern/vgg.ipynb:383]: the same as above

File: jupyter/d2l-java/chapter_convolutional-modern/vgg.ipynb (hunk 1 above)
Comment (aksrajvanshi) [reply jupyter/d2l-java/chapter_convolutional-modern/vgg.ipynb]: fixed in both ipynbs. Thanks!

File: jupyter/d2l-java/chapter_convolutional-modern/alexnet.ipynb [hunk 3]
```diff
@@ -332,37 +332,43 @@
...
     "import ai.djl.nn.SequentialBlock;\n",
-    "import ai.djl.nn.convolutional.Conv2D;\n",
+    "import ai.djl.nn.convolutional.Conv2d;\n",
     "import ai.djl.nn.core.Linear;\n",
+    "import ai.djl.nn.norm.Dropout;\n",
     "import ai.djl.nn.pooling.Pool;\n",
     "import ai.djl.training.DefaultTrainingConfig;\n",
-    "import ai.djl.training.EasyTrain;\n",
-    "import ai.djl.nn.norm.Dropout;\n",
+    "import ai.djl.training.GradientCollector;\n",
```
Comment (stu1130) [inline jupyter/d2l-java/chapter_convolutional-modern/alexnet.ipynb:357]: nit: where do we use this?

File: jupyter/d2l-java/chapter_convolutional-modern/alexnet.ipynb [hunk 4]
```diff
@@ -332,37 +332,43 @@
...
+    "import ai.djl.training.ParameterStore;\n",
```
Comment (stu1130) [inline jupyter/d2l-java/chapter_convolutional-modern/alexnet.ipynb:358]: see above

**Pull request merged**